prov/gni/include/gnix.h

```diff
@@ -466,6 +471,8 @@ struct gnix_fid_ep {
 	bool tx_enabled;
 	bool rx_enabled;
 	bool requires_lock;
+	int last_cached;
+	struct gnix_addr_cache_entry addr_cache[5];
```

Comment:
What's 5 based on?  Also, 5 is used in more than one place so in the very least it should be a #define possibly configurable via open_ops